postgres/migrations/V1.27__job_applications_table.sql

```diff
@@ -2,10 +2,32 @@
 DROP TABLE IF EXISTS job_applications;
 CREATE TABLE job_applications (
 	application_id serial PRIMARY KEY,
+	date_added timestamp with time zone default now(),
```

Comment:
Migrations should never be edited once they have been merged onto master. Editing them will break the migration system. Use a new migration with alter table statements instead. For example:
```pgsql
ALTER TABLE job_applications ADD COLUMN date_added timestamp with time zone default now();
```

Reply:
This still is not resolved. There should not be any edits to this file.

Reply:
For time's sake I went ahead and moved these column additions to the new migration.